Added board 93 from WO #16

Cupcakus opened the pull request.

derimmer commented:
Great to see a new board builder. I have taken a quick look at the board and it seems fine and the LOS checker works well on the board image. Although the one building hex is kind of a non-standard look and I would suggest fixing that before release.

As well the board appears to be lacking some of the standard overlays that are included in all other MMP geoboards to handle terrain transitions such as Light Woods, various PTO changes, Orchard-Shellhole-Crag transforms, etc. @Cupcakus could you add those overlays before we release the board? Take a look at a few other boards and you will quickly see the standard ones that each of them have. 

Given the expectations of users, I would favour holding off adding this board to VASL until we can address the above. Other POV?

geezer09 commented:
I second what @derimmer said. I will only add that you should not hesitate to reach out if you run into any issues and need assistance. 

Thank you for your contribution.

tqrtuomo commented:
Agreed - this is close but there's a few small things that need cleaning up:
* The building needs to use standard building colors
* The edges of some features are blurred, which introduces colors that VASL doesn't know what to do with

@Cupcakus - if you ping me at tuomorepetti@gmail.com, I'd be glad to walk you through these things or share some resources that might help.

Cupcakus commented:
Thanks for the feedback everyone...

Yes, since I built it just for myself to play WO #16 I didn't bother with the PTO or other Terrain SSR overlays the other boards have, shouldn't be a big problem to add them.

I couldn't find an example of the one building in the map in the resources I have, so it's lifted directly from the board scan and cleaned up a bit.  The colors don't match... I fixed this by running the LOS tool with an alternate image of the building being all one color.

As for the edges being blurred... Photoshop just does this occasionally, even if I disable anti-aliasing, it still blurs edges all the time, would be useful to have some tips here.  I'll contact @tqrtuomo and submit an updated PR in a couple weeks.

derimmer commented:
Super and looking forward to getting the revised board.

derimmer commented:
bd93 is in the system and functioning well. The necessary files are in the github repo both in bdFiles and src. 

No further action required; closing the PR
